open-ce/build_env.py

```diff
@@ -68,6 +69,29 @@ def make_parser():
 
     return parser
 
+def _run_tests(build_tree, conda_env_files):
+    """
+    Run through all of the tests within a build tree for the given conda environment files.
+
+    Args:
+        build_tree (BuildTree): The build tree containing the tests
+        conda_env_files (dict): A dictionary where the key is a variant string and the value
+                                is the name of a conda environment file.
+    """
+    failed_tests = []
+    # Run test commands for each conda environment that was generated
+    for variant_string, conda_env_file in conda_env_files.items():
+        test_commands = build_tree.get_test_commands(variant_string)
+        if test_commands:
+            print("\n*** Running tests within the " + conda_env_file + " conda environment ***\n")
```

Comment:
Is `conda_env_file` absolute path of the conda env file? If yes, then this statement probably needs correction.